tst/DataChannelFunctionalityTest.cpp

```diff
@@ -92,6 +92,78 @@ TEST_F(DataChannelFunctionalityTest, createDataChannel_Disconnected)
     ASSERT_EQ(1u, remoteOpen.channels.at("Answer PeerConnection"));
 }
 
+TEST_F(DataChannelFunctionalityTest, dataChannelSendRecvMessageAfterDtlsCompleted)
+{
+    RtcConfiguration configuration;
+    PRtcPeerConnection offerPc = NULL, answerPc = NULL;
+    PRtcDataChannel pOfferDataChannel = nullptr, pAnswerDataChannel = nullptr;
+    SIZE_T pOfferRemoteDataChannel = 0, pAnswerRemoteDataChannel = 0;
```

Comment:
Naming: these aren't pointers - suggest dropping 'p' prefix

Reply:
they are actually storing pointers. using size_t because of atomic operation.

Reply:
yeah, understood that, though the type is size-t; really a nit